docs/reference/tags/form_tags/create_line_item/create_line_item.md

```diff
@@ -63,7 +71,7 @@ The following example redirects to the homepage.
 {% raw %}
 ```liquid
 {% form "create_line_item", return_to: '/home' %}
-      <input type="hidden" data-variant-id="{{item_id}}" name="items[][variant_id]" value="{{item_id}}"/>
+      <input type="hidden" name="items[][variant_id]" value="{{item_id}}"/>
```

Comment:
since working on the bingo loco yesterday my one suggestion for the docs is to be more specific than item_id. The value should be the product.id? variant.id?

Reply:
It should be variant.id or extra.id - I've made that clearer now